spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -35,20 +37,27 @@
     let!(:two) do
       create(:generic_work, creator: ["Fred"], title: ["abc"], language: ['en'], user: user)
     end
+
+    let!(:three) do
+      create(:generic_work, creator: ["Fred"], title: ["abc"], language: ['en'])
```

Comment:
On line 13 above you are using `create(:work,...)`  can we stick with that for consistency?